vidify/player/web.py

```diff
@@ -0,0 +1,83 @@
+"""
+"""
+
+import logging
+from typing import Optional
+from threading import Thread
+from webbrowser import open
+from threading import Timer, Thread
+
+try:
+    import flask
+except OSError:
+    raise ImportError("Flask is not installed")
+
+from vidify import Platform, CURRENT_PLATFORM
+from vidify.player.generic import PlayerBase
+from PyQt5.QtCore import QObject
+import sys
+import os
+
+PORT = 9999
+ROOT_URL = 'http://localhost:{}/video'.format(PORT)
+DEFAULT_VIDEO_ID = "GfKs8oNP9m8"
+class WebPlayer(PlayerBase, QObject):
+    DIRECT_URL = False
+    app = None
+    currentMedia = ""
+    isPlaying = True
+    def __init__(self, vlc_args: Optional[str] = None) -> None:
```

Comment:
You can get rid of the `vlc_args` here. I know, the player initialization is a bit confusing. In #108 I modified so that it just passes a `config` object. You can remove the argument for now and I'll fix it after I merge #108.